test/checkpoint/test_checkpoint.c

```diff
@@ -240,6 +264,34 @@ main(int argc, char *argv[])
 
     testutil_work_dir_from_path(g.home, 512, (&g.opts)->home);
 
+    /* 
+     * Always preserve home directory. Some tests rely on the
+     * home directory being present to compare results between runs.
+     */
+    g.opts.preserve = true;
```

Comment:
I haven't seen this being used in test/checkpoint before, why has this become relevant now?